client/components/AvailabilityGrid/AvailabilityGrid.js

```diff
@@ -227,60 +229,79 @@ class AvailabilityGrid extends React.Component {
   handleCellMouseDown(ev) {
     if (this.props.heatmap) return;
 
-    let { startSelection } = this.state;
     const {
-      rangeSelected,
-    } = this.state;
-    const cellBackground = getComputedStyle(ev.target)['background-color'];
-    const cellIsSelected = cellBackground !== 'rgba(0, 0, 0, 0)';
-
-    if (!startSelection) {
-      this.updateCellAvailability(ev);
-    } else {
-      const {
         generateRange,
         updateAvailabilityForRange,
-      } = this.constructor;
-
-      let updateAvail;
-      if (rangeSelected) {
-        updateAvail = this.constructor.removeCellFromAvailability;
-      } else {
-        updateAvail = this.constructor.addCellToAvailability;
-      }
+    } = this.constructor;
 
-      const thisRow = Number(ev.target.getAttribute('data-row'));
-      const thisCol = Number(ev.target.getAttribute('data-col'));
-      const initialRow = this.state.mouseDownRow;
-      const initialCol = this.state.mouseDownCol;
+    const thisRow = Number(ev.target.getAttribute('data-row'));
+    const thisCol = Number(ev.target.getAttribute('data-col'));
 
-      const rowRange = generateRange(thisRow, initialRow);
-      const colRange = generateRange(thisCol, initialCol);
+    const cellBackgroundColor = getComputedStyle(ev.target)['background-color'];
+    const cellIsSelected = (cellBackgroundColor === "rgb(128, 0, 128)");
 
-      updateAvailabilityForRange(rowRange, colRange, updateAvail);
+    let updateAvail;
+    if (cellIsSelected)
+      updateAvail = this.constructor.removeCellFromAvailability;
```

Comment:
use brackets for all if statements: **if(condition) {
resolution 
}  else {
}**